src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -1065,7 +1025,7 @@ def run_dm_activities_with_ior(self, tool, create_dataset=False, pool=None, cont
             # create initial datasets
             if not pool:
                 pool = self.create_pool()
-            cont = self.create_cont(pool, oclass=self.ior_cmd.dfs_oclass.value)
+            cont = self.add_container(pool, oclass=self.ior_cmd.dfs_oclass.value)
```

Comment:
(pylint-assignment-from-no-return) Assigning result of a function call, where the function has no return
(pylint-unexpected-keyword-arg) Unexpected keyword argument 'oclass' in method call